<?xml version="1.0" encoding="UTF-8"?>

<launch>
  <rosparam file="$(find eurobench_benchmark_core)/config/general.yaml" command="load" subst_value="true" />

  <node name="eurobench_benchmark_core" pkg="eurobench_benchmark_core" type="benchmark_core.py" output="screen">
    <param name="benchmark_group" value="BEAST" />
  </node>
  <include file="$(find benchmark_gui)/launch/beast_gui.launch" />

</launch>
